pom.xml

```diff
@@ -49,14 +49,13 @@
       <version>5.2-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
+
+    <!-- XWiki -->
     <dependency>
-      <groupId>com.celements</groupId>
-      <artifactId>celements-component</artifactId>
-      <version>5.1-SNAPSHOT</version>
```

Comment:
do not remove `celements-component`, see https://github.com/celements/celements-xwiki/pull/150

Reply:
readded the depdency due to failing builds

Reply:
tha.